odoo/addons/base/models/ir_sequence.py

```diff
@@ -150,14 +150,15 @@ def _get_current_sequence(self, sequence_date=None):
     use_date_range = fields.Boolean(string='Use subsequences per date_range')
     date_range_ids = fields.One2many('ir.sequence.date_range', 'sequence_id', string='Subsequences')
 
-    @api.model
-    def create(self, values):
+    @api.model_create_multi
+    def create(self, vals_list):
         """ Create a sequence, in implementation == standard a fast gaps-allowed PostgreSQL sequence is used.
         """
-        seq = super(IrSequence, self).create(values)
-        if values.get('implementation', 'standard') == 'standard':
-            _create_sequence(self._cr, "ir_sequence_%03d" % seq.id, values.get('number_increment', 1), values.get('number_next', 1))
-        return seq
+        seqs = super().create(vals_list)
+        for seq, vals in zip(seqs, vals_list):
+            if seq.implementation == 'standard':
+                _create_sequence(self._cr, "ir_sequence_%03d" % seq.id, vals.get('number_increment', 1), vals.get('number_next', 1))
```

Comment:
Note that you could do like implementation field, and use directly seq.number_increment and seq.number_next. This allows to stop using vals_list and make code easier to read as it becomes a simple loop on sequences :) .